src/FamilyHubs.ServiceDirectory.Web/web.Release.config

```diff
@@ -0,0 +1,16 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
is there any otherway to do this, this file wont be picked up in production if the aspnet_environment vatiable is not set to Release

Reply:
if this is the only way force read this file in the startup.cs

Reply:
I've just checked and `ASPNETCORE_ENVIRONMENT` is set to `Release` in prod.

To remove the `x-powered-by` header, as far as I know, needs the appropriate section in the `web.config` file. The `web.config` file with Azure App Services is generated at publish time, so it can't be added directly.

I'm open to doing it a different way if you know of one (doing it in code, like you can with the `server` header, would be preferable).